### DISCUSSION TITLE
How to avoid slow motion sound?

### DISCUSSION THREAD

**Discussion by BurhanMuhyiddin** [coqui-ai/TTS#1659] [General Q&A]
Hi all. I have recently discovered coqui TTS and tested it. I found it very interesting. However, I have a small problem. While I can synthesis the sound and get .wav file, after the clear sound, sometimes there is also slow motion version of it. How I can avoid that slowed motion version? I just need the normal one. Thank you beforehand for your time.

**Comment by p0p4k** [comment]
play around with `synthesizer.tts_model.length_scale` parameter. It controls the speed.

**Reply by BurhanMuhyiddin** [reply]
Thank you for your answer, but for Tacotron-2 DDC pretrained model, this variable is unavailable.

**Reply by p0p4k** [reply]
Yes, unfortunately it is not available for Tacotron-2.

**Reply by Ap1075** [reply]
Is there a way to deterministically know the affect of length_speed in the rate of speech?

**Reply by zzzgit** [reply]
> Yes, unfortunately it is not available for Tacotron-2.

`vits` as well.

I think `TTS`'s documentation should tell us more about which models support it and which do not.
